Dockerfile.assisted-service-build

```diff
@@ -19,4 +19,5 @@ RUN chmod g+xw -R $GOPATH && chmod g+xw -R $(go env GOROOT)
 
 COPY --from=quay.io/openshift/origin-cli:latest /usr/bin/oc /usr/bin
 COPY --from=quay.io/operator-framework/upstream-opm-builder:v1.16.1 /bin/opm /bin
+COPY ./data /tmp/data
```

Comment:
It needs it otherwise we get the below error when executing the below command in skipper shell
```
bash-4.2$ make ci-unit-test
/bin/sh: /tmp/data/default_must_gather_versions.json: No such file or directory
cat: /tmp/data/default_hw_requirements.json: No such file or directory
/bin/sh: /tmp/data/default_release_images.json: No such file or directory
/bin/sh: /tmp/data/default_release_images.json: No such file or directory
/bin/sh: /tmp/data/default_os_images.json: No such file or directory
/bin/sh: /tmp/data/default_os_images.json: No such file or directory
```